tests/unit/components/paper-toast-test.js

```diff
@@ -0,0 +1,35 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
This should be converted to an integration test.
This is the test that is failing (`needs: ['service:transition-events']` no longer makes sense in the new ember-css-transitions version we're using).